nltk/metrics/distance.py

```diff
@@ -103,6 +103,76 @@ def edit_distance(s1, s2, substitution_cost=1, transpositions=False):
     return lev[len1][len2]
 
 
+def _edit_dist_backtrace(lev_map):
+    i, j = len(lev_map) - 1, len(lev_map[1]) - 1
+    alignment = [(i, j)]
+
+    while (i, j) != (0, 0):
+        # find direction of minimum cost
+        s1_skip_cost = s2_skip_cost = sub_cost = float('inf')
+        if i - 1 >= 0:
+            s1_skip_cost = lev_map[i - 1][j]
+        if j - 1 >= 0:
+            s2_skip_cost = lev_map[i][j - 1]
+        if i - 1 >= 0 and j - 1 >= 0:
+            sub_cost = lev_map[i - 1][j - 1]
+
+        # move in direction of minimum cost
+        if sub_cost <= s1_skip_cost and sub_cost <= s2_skip_cost:
+            i, j = i - 1, j - 1
+        elif s2_skip_cost <= s1_skip_cost:
+            i, j = i, j - 1
+        else:
+            i, j = i - 1, j
+
+        alignment.append((i, j))
+    alignment.reverse()
```

Comment:
You can combine this line with the next one by doing this:
```python
return reversed(alignment)
```

Reply:
Made the change and added list() around it to return a list rather than an iterator.